src/rviz/ogre_helpers/movable_text.cpp

```diff
@@ -106,12 +105,25 @@ void MovableText::setFontName(const String &fontName)
     Ogre::MaterialManager::getSingleton().remove(mName + "Material");
   }
 
-  if (mFontName != fontName || mpMaterial.isNull() || !mpFont)
+  if (mFontName != fontName || mpMaterial.isNull() || mpFont.isNull())
   {
     mFontName = fontName;
     mpFont
-        = (Font *) FontManager::getSingleton().getByName(mFontName).getPointer();
-    if (!mpFont)
+        = FontManager::getSingleton().getByName(mFontName);
+
+    // Workaround for getByName bug
```

Comment:
Could this be fixed by passing the correct resource group to `getByName` instead?

Reply:
Good idea, will test :)